olp-cpp-sdk-core/include/olp/core/client/OlpClientSettings.h

```diff
@@ -167,6 +171,11 @@ struct OlpClientSettings {
    * @brief The network handler.
    */
   boost::optional<NetworkAsyncHandler> network_async_handler = boost::none;
+
+  /**
+   * @brief The task scheduler instance.
```

Comment:
Please add also comment that if nullptr all requests will be performed syncronous.

Reply:
done